neurots/utils.py

```diff
@@ -41,8 +43,32 @@ def format_values(obj, decimals=None):
     return obj
 
 
-def _check(data):
-    """Checks if data in dictionary are empty."""
-    for key, val in data.items():
-        if len(val) == 0:
-            raise NeuroTSError(f"Empty distribution for diameter key: {key}")
+def convert_from_legacy_neurite_type(data):
```

Comment:
I think we should also raise a warning in this function when a legacy key is transformed, so users know they have to change their inputs before the next major version.

Reply:
yes, done!